src/BDCS/API/V0.hs

```diff
@@ -712,7 +712,7 @@ recipesTag repoLock branch recipe_name = liftIO $ RWL.withRead (gitRepoLock repo
 
 
 -- | JSON response for /recipes/diff
-data RecipesDiffResponse = RecipesDiffResponse {
+newtype RecipesDiffResponse = RecipesDiffResponse {
```

Comment:
I don't like this hlint warning for the returned JSON -- I'd rather keep them all consistent as data. And in the future we may add more fields to them.